.NET/Samples/SimpleConsole/Program.cs

```diff
@@ -15,7 +15,7 @@ namespace SimpleConsole
     class Program
     {
         // Use English for the Recognizers culture
-        private const string DefaultCulture = Culture.English;
+        private const string DefaultCulture = Culture.Dutch;
```

Comment:
Be careful not to change the culture in the Sample.

Reply:
#resolved